app/common/db_utils.py

```diff
@@ -0,0 +1,12 @@
+import os
+import urllib
+
+from sqlalchemy import create_engine
+
+
+def create_sqlalchemy_engine(**kwargs):
+    DB_CREDENTIALS = os.environ.get("DATABASE_PARAMS")
```

Comment:
Do I remember correctly that we are unable to use config constant `DevelopmentConfig.SQLALCHEMY_DATABASE_URI`  due to a lack of config in the context?

Reply:
Technically we could use `DevelopmentConfig` here, but this function is used by the `add_lrf_table` migration and by `build_localised_acyclic_graph`, so it would be misleading to use `DevelopmentConfig` here to get `DATABASE_PARAMS` indirectly.